app/home/controllers.js

```diff
@@ -1,8 +1,10 @@
 const dateHelpers = require('../../common/helpers/date')
+const contentfulService = require('../../common/services/contentful')
 
-function dashboard(req, res) {
+async function dashboard(req, res) {
   const currentWeek = dateHelpers.getCurrentWeekAsRange()
   const today = new Date().toISOString()
+  const content = await contentfulService.fetchEntries()
```

Comment:
```suggestion
  const whatsNewContent = await contentfulService.fetchEntries()
```

I'm thinking this might make it a little clearer which content the variable contains.